samples/scala-valueentity-customer-registry/src/main/resources/logback.xml

```diff
@@ -6,7 +6,7 @@
                 <timestampFormatTimezoneId>Etc/UTC</timestampFormatTimezoneId>
 
                 <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter">
-                    <prettyPrint>true</prettyPrint>
+                    <prettyPrint>false</prettyPrint>
```

Comment:
Maybe add a comment that this is (currently) mandatory to get pretty logging in the Akka Serverless UI?